extract_msg/__init__.py

```diff
@@ -28,7 +28,7 @@
 
 __author__ = 'Matthew Walker & The Elemental of Creation'
 __date__ = '2018-12-05'
```

Comment:
if we are going to have a release date let's update it

Reply:
Bumped the date